lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -9,18 +9,16 @@
 	/**
 	 * Whether a theme or its parent have a theme.json file.
 	 *
-	 * @param boolean $clear_cache Whether the cache should be cleared and theme support recomputed. Default is false.
+	 * The result would be cached via the WP_Object_Cache
```

Comment:
Does the cache key become a public key we need to document, or rather this should document the method for clearing it? The method may affect other caches as well. cc @spacedmonkey @felixarntz

Reply:
Not that I know.

Reply:
Documented the public method to clean this.